Dockerfile.tests

```diff
@@ -0,0 +1,20 @@
+FROM registry.centos.org/centos/centos:7
+
+ENV F8A_WORKER_VERSION=561636c
+
+RUN yum install -y epel-release &&\
+    yum install -y gcc git python34-pip python34-requests httpd httpd-devel python34-devel &&\
+    yum clean all
+
+COPY ./requirements.txt /
+
+RUN pip3 install -r requirements.txt && rm requirements.txt
+
+RUN pip3 install git+https://github.com/fabric8-analytics/fabric8-analytics-worker.git@${F8A_WORKER_VERSION}
```

Comment:
We don't need to install these dependencies if we inherit them from the base image.